Assets/Scripts/Input/Rewind/RewindMouseAndKeyboardInputControl.cs

```diff
@@ -80,15 +92,15 @@ void IInputControl.ConfigureInputEvents(PlayerInput playerInput)
         {
             // Subscribe all events from input control
             // Methods should be subscribed as the first in each event list
-            
+
             // Rewind
             playerInput.actions["endrewind"].performed += ((IRewindInputControl)this).OnEndRewind;
             playerInput.actions["scrub"].performed += ((IRewindInputControl)this).OnScrub;
             playerInput.actions["scrub"].canceled += ((IRewindInputControl)this).OnScrub;
-            
+
             // Menu Action
             playerInput.actions["pause"].performed += ((IRewindInputControl)this).OnPause;
-            
+
             // View Orientation
             playerInput.actions["rotatecamera"].performed += ((IRewindInputControl)this).OnRotateCamera;
             playerInput.actions["rotatecamera"].canceled += ((IRewindInputControl)this).OnRotateCamera;
```

Comment:
very minor, and not related to this PR, but could be worth consting these in the future

Reply:
Will be be done in seperate tech debt

Reply:
Ticket #57